lib/component.js

```diff
@@ -396,6 +412,34 @@ class Component extends WebComponent {
     return state;
   }
 
+  _updateAttrs(attr) {
+    const attrsSchema = this.constructor.attrsSchema;
+    if (attrsSchema.hasOwnProperty(attr)) {
+      const propName = attrsSchema[attr].propName || this._toCamelCase(attr);
```

Comment:
Haha! One person's slippery slope is another person's water slide.

I personally prefer camel case because it reads better in template and has consistent access. i.e  suppose you have `progressive`, `sections`, `empty-message` and `search-filter`. I know its minor but I'm thinking as dev experience.

Option 1) They are accessed as `$attrs.progressive`, `$attrs.sections`, `$attrs['empty-message']`, `$attrs['search-filter]`.

Option 2) They are accessed as `$attrs.progressive`, `$attrs.sections`, `$attrs.emptyMessage`, `$attrs.searchFilter`.

Also dom element.style already give us a camel cased access so it's not a new thing. Plus with the auto camel casing, we have an enforced convention.

Let me poll others and see what they think.

Reply:
Option 3 they are accessed as `$attrs['progressive']`, `$attrs['sections']`, `$attrs['empty-message']`, `$attrs['search-filter']` (which most closely matches the standard `getAttribute('myattr')` pattern which we already use)